tests/inspectdb/tests.py

```diff
@@ -101,10 +101,8 @@ def test_number_field_types(self):
         assertFieldType('bool_field', "models.{}()".format(bool_field_type))
         assertFieldType('null_bool_field', 'models.{}(blank=True, null=True)'.format(bool_field_type))
 
-        if connection.features.can_introspect_decimal_field:
-            # django-spanner creates DecimalField as FLOAT64 until Spanner
-            # supports numeric.
-            assertFieldType('decimal_field', "models.FloatField()")
+        if connection.vendor != 'sqlite':
```

Comment:
If we just rolled back the relevant hunk in d18d92245a4 this line wouldn't change, but it looks like this line comes from https://github.com/django/django/commit/e198beadad56d5fdd06756b097446b7046a84010 instead. I think this works, but this is now another change we need to cherry-pick if we update django.

Reply:
yes, this is added issue to maintain.